src/bt_auto_dumper/_v2/__main__.py

```diff
@@ -20,22 +20,22 @@ def main(apiver: str | None = None):
     apiver = apiver or pathlib.Path(__file__).parent.name
     parser = argparse.ArgumentParser(description=f"BT Auto Dumper CLI {apiver}")
     parser.add_argument("--note", help="Comment or note for the operation", type=str, default="")
-    parser.add_argument("subnet_identifier", help="Subnet Identifier", type=str)
-    parser.add_argument("autovalidator_address", help="AutoValidator Address", type=str)
+    parser.add_argument("subnet_identifier", help="Subnet Identifier", type=str, default="")
+    parser.add_argument("autovalidator_address", help="AutoValidator Address", type=str, default="")
     parser.add_argument(
         "subnet_realm",
         help="Subnet Realm",
         type=str,
         choices=["testnet", "mainnet", "devnet"],
         default="mainnet",
     )
-    parser.add_argument("--set-autovalidator-address", help="Set a new autovalidator address", type=str)
-    parser.add_argument("--set-codename", help="Set a new Subnet Identifier codename", type=str)
+    parser.add_argument("--set-autovalidator-address", help="Set a new autovalidator address", type=str, default="")
+    parser.add_argument("--set-codename", help="Set a new Subnet Identifier codename", type=str, default="")
```

Comment:
In the current implementation if you call it twice, once with --set-autovalidator-address and the second time with --set-codename, it will overwrite the config and it won't be set properly. Please fix that.

Reply:
![image](https://github.com/user-attachments/assets/b0c65287-fd3e-45f2-90d5-e15332095189)
in the update_config function, the set-autovalidator-address, set-codename is only set when the variables have non-value as you can see in the above image.